.github/workflows/upgrade-typescript-api.yml

```diff
@@ -0,0 +1,32 @@
+name: Upgrade typescript API
+on:
+  workflow_dispatch:
+    inputs:
+      spec_version:
+        description: runtime spec version
+        required: true
+
+jobs:
+  create-tracing-runtime:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          ref: runtime-${{ github.event.inputs.spec_version }}
+      - name: Use Node.js 14.x
+        uses: actions/setup-node@v2
+        with:
+          node-version: 14.x
```

Comment:
Not a big deal but I think is better to use the current LTS version which is 16.x.

Reply:
Yes I agree, I put 14 because it's the version of node used in all our GH workflows, I think we'll have to upgrade them all at the same time, I just created a JIRA task for that :)

Reply:
We are now using 18.x on our CI but it is good ot use 16 here

Reply:
@crystalin in all our GH workflows we explicitly use node 14.x

Reply:
Not anymore for tests, because they run into docker: 
https://github.com/PureStake/moonbeam/blob/master/.github/workflows/build.yml#L211
https://github.com/PureStake/moonbeam/blob/master/docker/polkadot-para-tests.Dockerfile#L5
It takes the latest node (18)